crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -13,20 +13,17 @@ use diesel::{
     ExpressionMethods, PgConnection, QueryDsl, RunQueryDsl,
 };
 
-use service_utils::{
-    bad_argument,
-    helpers::{construct_request_headers, generate_snowflake_id, request},
-    response_error,
-    result::{self as superposition, AppError},
-    unexpected_error,
-};
-
-use superposition_types::{SuperpositionUser, User};
+use service_utils::helpers::{construct_request_headers, generate_snowflake_id, request};
 
 use reqwest::{Method, Response, StatusCode};
 use service_utils::service::types::{
     AppHeader, AppState, CustomHeaders, DbConnection, Tenant,
 };
+use superposition_macros::{bad_argument, response_error, unexpected_error};
+use superposition_types::{
+    result::{self as superposition, AppError},
```

Comment:
Here too, can you remove import of result::AppError as it is already being included when we import result as superposition, and also make the necessary changes in the code.